bentoml/frameworks/evalml.py

```diff
@@ -0,0 +1,106 @@
+import os
+import pickle
+
+from bentoml.exceptions import ArtifactLoadingException, MissingDependencyException
+from bentoml.service.artifacts import BentoServiceArtifact
+from bentoml.service.artifacts.common import PickleArtifact
+
+class EvalMLModelArtifact(BentoServiceArtifact):
+    """
+    Abstraction for saving/loading EvalML models
+
+    Args:
+        name (str): Name for the artifact
+        pickle_extension (str): The extension format for pickled file
+        pickle_module (str, module): pickle format to use: 'pickle' or 'cloudpickle'
```

Comment:
I saw that `PickleArtifact` supports both python native `pickle` and `cloudpickle`. So I wanted to give users the option here of using either format. This makes testing slightly more complex. I would gladly rip this out and stick to support for just one format if that is preferable to reviewers.